examples/controlled/README.md

```diff
@@ -0,0 +1,60 @@
+# Controlled
+
+This example shows how to provide some initial focal point values.
```

Comment:
In this example you can check how to set an initial focal point value (by default it is show centered, but you can overrride it)

Reply:
Done